src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -383,4 +389,44 @@
      */
     List<DimensionType> getDimensionTypes();
 
+    // TODO: Find a better place for these methods
+
+    /**
+     * Loads a {@link Favicon} from the specified encoded string. The format of
+     * the input depends on the implementation.
+     *
+     * @param raw The encoded favicon
+     * @return The loaded favicon
+     * @throws IOException If the favicon couldn't be loaded
+     */
+    Favicon loadFavicon(String raw) throws IOException;
+
+    /**
+     * Loads a favicon from a specified {@link File}.
+     *
+     * @param file The favicon file
+     * @return The loaded favicon from the file
+     * @throws IOException If the favicon couldn't be loaded
+     * @throws FileNotFoundException If the file doesn't exist
+     */
+    Favicon loadFavicon(File file) throws IOException;
```

Comment:
This is already included in `IOException` (`FileNotFoundException` is a subclass of `IOException`) and IntelliJ IDEA doesn't stop with complaining that it is already defined.